run.py

```diff
@@ -46,62 +44,45 @@ def cd(path):
     yield
     os.chdir(old_dir)
 
-def setup_sasmodels():
-    """
-    Prepare sasmodels for running within sasview.
-    """
-    # Set SAS_MODELPATH so sasmodels can find our custom models
-
-    from sas.system.user import get_user_dir
-    plugin_dir = os.path.join(get_user_dir(), PLUGIN_MODEL_DIR)
-    os.environ['SAS_MODELPATH'] = plugin_dir
-
 def prepare():
     # Don't create *.pyc files
     sys.dont_write_bytecode = True
 
-    # find the directories for the source and build
-    from distutils.util import get_platform
-    root = abspath(dirname(realpath(__file__)))
-
-    platform = '%s-%s' % (get_platform(), sys.version[:3])
-    build_path = joinpath(root, 'build', 'lib.' + platform)
+    # Turn numpy warnings into errors
+    #import numpy; numpy.seterr(all='raise')
 
-    # Notify the help menu that the Sphinx documentation is in a different
-    # place than it otherwise would be.
-    os.environ['SASVIEW_DOC_PATH'] = joinpath(build_path, "doc")
-
-    try:
-        import periodictable
-    except ImportError:
-        addpath(joinpath(root, '..', 'periodictable'))
+    # Find the directories for the source and build
+    root = abspath(dirname(realpath(__file__)))
 
-    try:
-        import bumps
-    except ImportError:
-        addpath(joinpath(root, '..', 'bumps'))
+    # TODO: Do we prioritize the sister repo or the installed package?
```

Comment:
Out of curiosity, where/when is PYTHONPATH set. I don't have it set...

Reply:
@smk78, `run.py` is full of calls to `addpath` which is a function that is fiddling with `PYTHONPATH` (and/or `sys,path`). The custom test runners used to have lots of that too and they have been slowly retired, which helps.

Reply:
The point of run.py is to set up the python when running in development mode. I often set up parallel directories for different branches and don't want to set up a complete python environment when I do so. This will be increasingly the case if we go to a fork rather than branch workflow for PRs.

I would think you will always want the equivalent for testing. How else do you run the tests before installing?